CHANGELOG.md

```diff
@@ -17,7 +17,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Internal Changes
 
--
+- Documented support for evaluating rules against default `KubeletConfig` objects.
+  Please refer to the [usage guide](./doc/usage.md) for more information on this feature.
```

Comment:
Since documentation can be considered external, I'd consider moving this up to the 'Enhancements' section.